src/payment/PaymentPage.jsx

```diff
@@ -30,6 +30,7 @@ class PaymentPage extends React.Component {
         isNumEnrolledExperiment = false,
         REV1045Experiment = false,
         isPriceMessageExperiment = false,
+        isTransparentPricingExperiment = false,
```

Comment:
not sure how much work it would be or if it would be feasible, but imo it would be better if we stopped hardcoding the variables for each experiment and just checked the experimentVariables dictionary directly

Reply:
@MatthewPiatetsky I agree, or even do an experiment clean up and remove the experiments that did not reach significance/not planned to be used again. To answer your other questions - 
1. I've added the mobile view on the description. 
2. We're not translating for this test, it will be for all learners. 
3. Came from product, not sure how they got to that number.

Reply:
would it still work if you didn't hardcode the isTransparentPricingExperiment variable and just passed through the experimentVariables dict?